package.json

```diff
@@ -4,27 +4,31 @@
         "format": "npx prettier --write ."
     },
     "lint-staged": {
-        "**/*.{js,jsx,ts,tsx,json,css,scss,md}": "prettier --write"
+        "**/*.{js,jsx,ts,tsx,json,css,scss}": "prettier --write"
     },
     "dependencies": {
+        "@ai-sdk/anthropic": "^0.0.19",
```

Comment:
Nit: doesn't look like anthropic is actually used.

Reply:
It's wired up in the model config for claude-3-haiku! Haven't tested it yet but will soon